src/helpers/addressHelper.js

```diff
@@ -0,0 +1,24 @@
+export const formatAddress = (address) => {
+  if (!address) return;
+
+  const { city, street, zip, addition } = address;
+  let readableAddress = '';
+
+  if (!city && !street && !zip && !addition) return null;
+
+  // build the address in multiple steps to check every data before rendering
+  if (addition) {
+    readableAddress += `${addition}${'\n'}`;
+  }
+  if (street) {
+    readableAddress += `${street},${'\n'}`;
+  }
+  if (zip) {
+    address += `${zip} `;
```

Comment:
`address` -> `readableAddress`

Reply:
done in 3795523c9fe7f462df6678bc1c684c0c58969914